src/expr/term_canonize.cpp

```diff
@@ -105,7 +105,7 @@ Node TermCanonize::getCanonicalFreeVar(TypeNode tn, unsigned i, uint32_t tc)
   while (tvars.size() <= i)
   {
     std::stringstream oss;
-    oss << tn;
```

Comment:
Can you perhaps add "fun_" or something to the name here?

Reply:
Better, can you make the name simply `f`?

Reply:
You mean for all functional-type vars to have the names as `f` ++ `i`?

Reply:
Yes

Reply:
OK, done